build.sbt

```diff
@@ -70,9 +73,6 @@ libraryDependencies ++= {
     "org.scalatest"       %%  "scalatest"     % "2.2.6"   % "test",
     "org.mock-server"      %  "mockserver-netty" % "3.10.2" % "test",
     "org.everit.json"      %  "org.everit.json.schema" % "1.4.0" % "test"
-    // -- Logging --
-    ,"ch.qos.logback" % "logback-classic" % "1.1.3"
-    ,"com.typesafe.scala-logging" %% "scala-logging-slf4j" % "2.1.2"
```

Comment:
I didn't quite understand it either, but this fixed things - would love to see if other people encounter any issues.